src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -40,7 +40,7 @@ public class CertificateGenerator
         // Give the cert a grace period in case there's a time skew between machines
         private readonly TimeSpan _gracePeriod = TimeSpan.FromHours(1);
 
-        private const string _authorityCanonicalName = "DO_NOT_TRUST_WcfBridgeRootCA";
+        private const string _authorityCanonicalName = Constants.CertificateIssuerName;
         private const string _signatureAlthorithm = "SHA1WithRSAEncryption";
```

Comment:
Unless @iamjasonp objects, I would prefer to rename the const in Constants something like CertificateAuthorityName and just use it everywhere (and not have one const here point to the other const in Constants.